assets/js/script.js

```diff
@@ -420,21 +419,34 @@ var displayWordData = function (wordObjArr) {
                     resultBody.append(noAudio);
                 };
 
+                //Display Image fetched from Pexels
+                var pexelsPhoto = document.createElement('img');
+                pexelsPhoto.setAttribute('src', wordData.image_m);
+                resultBody.append(pexelsPhoto);
+
                 // Get the modal
                 var modal = document.getElementById("myModal");
 
-                // Use 'getElementById' to get the ID of where the Img will be displayed
-                var picBodyEl = document.getElementById('img-body');
-
-                // Use 'getElementById' to get the ID of where the photographer name will be displayed
-                var photographerEl = document.getElementById("ph-body");
-                photographerEl.setAttribute('src', wordData.photographer);
+                // create an array for images only after fetch was called
```

Comment:
By creating an image array inside a for-loop, the result will only yield image[i] - a single image.